src/helpers.js

```diff
@@ -65,52 +78,106 @@ async function getCommerceAdminConfig() {
  */
 async function getAuthorizedOrganizations() {
 	const { consoleCLI } = await getLibConsoleCLI();
+
 	aioConsoleLogger.debug('Get the selected organization');
-	const key = CONSOLE_CONFIG_KEYS.ORG;
-	this.configOrgCode = Config.get(`${CONSOLE_CONFIG_KEYS.CONSOLE}.${key}`);
-	if (!this.configOrgCode) {
+
+	const consoleConfig = Config.get('console.org');
+
+	if (!consoleConfig) {
 		const organizations = await consoleCLI.getOrganizations();
 		const selectedOrg = await consoleCLI.promptForSelectOrganization(organizations);
+
 		aioConsoleLogger.debug('Set the console config');
-		Config.set(`${CONSOLE_CONFIG_KEYS.CONSOLE}.${key}`, {
+
+		Config.set('console.org', {
 			id: selectedOrg.id,
 			code: selectedOrg.code,
 			name: selectedOrg.name,
 		});
-		this.imsOrgCode = selectedOrg.code;
-		return { imsOrgCode: this.imsOrgCode };
+
+		return Object.assign({}, selectedOrg);
 	} else {
-		logger.info(`Selecting your organization as: ${this.configOrgCode.name}`);
-		return { imsOrgCode: this.configOrgCode.code };
+		logger.info(`Selecting your organization as: ${consoleConfig.name}`);
+
+		return Object.assign({}, consoleConfig);
 	}
 }
+
+async function getProjects(imsOrgId, imsOrgTitle) {
+	logger.info(`Initializing project selection for ${imsOrgId}`);
+
+	const { consoleCLI } = await getLibConsoleCLI();
+
+	const projects = await consoleCLI.getProjects(imsOrgId);
+	if (projects.length !== 0) {
+		const selectedProject = await consoleCLI.promptForSelectProject(projects);
+
+		return selectedProject;
+	} else {
+		aioConsoleLogger.error(`No projects found for the selected organization: ${imsOrgTitle}`);
+	}
+}
+
+async function getWorkspaces(orgId, projectId, imsOrgTitle, projectTitle) {
```

Comment:
The function names are plural and suggest that more than 1 org/project/workspace will be returned. Thoughts on renaming these to `getSelectedOrganization`, `getSelectedProject` and `getSelectedWorkspace` respectively?